libs/src/data-contracts/ABIs/UserStorage.json

```diff
@@ -98,68 +135,46 @@
       "constant": true,
       "inputs": [
         {
+          "internalType": "uint256",
           "name": "",
-          "type": "bytes16"
+          "type": "uint256"
         }
       ],
-      "name": "handlesTaken",
+      "name": "userHandles",
```

Comment:
I believe the checked in ABIs are outdated.

see
https://github.com/AudiusProject/audius-protocol/search?q=handlesTaken&type=

vs.

https://github.com/AudiusProject/audius-protocol/search?q=userHandles